systems/plants/RigidBodyManipulator.cpp

```diff
@@ -962,18 +977,17 @@ void RigidBodyManipulator::doKinematicsNew(double* q, bool compute_gradients, do
             dcrm(body.twist, joint_twist, body.dtwistdq, djoint_twistdq, &dcrm_twist_joint_twistdq); // TODO: make dcrm templated
             body.dJdotVdq = bodies[body.parent]->dJdotVdq
                 + dcrm_twist_joint_twistdq
-                + dTransformAdjoint(T, body.SdotV, body.dTdq_new, dSdotVdq);
+                + dTransformAdjoint(body.T_new, body.SdotV, body.dTdq_new, dSdotVdq);
 
             // dJdotvdv
             // TODO: exploit sparsity better
-            Matrix<double, TWIST_SIZE, Eigen::Dynamic> jacobian(TWIST_SIZE, 0);
             std::vector<int> v_indices;
-            geometricJacobian(0, i, 0, jacobian, &v_indices);
+            auto jacobian = geometricJacobian<double>(0, i, 0, 0, &v_indices);
```

Comment:
this might get us on windows?  i have the win64 build server humming, but have to fix the win32 build server.